src/FINDOLOGIC/Export/Data/Image.php

```diff
@@ -57,6 +57,15 @@ public function getUrl()
         return $this->url;
     }
 
+    private function setUrl($url)
+    {
+        $url = DataHelper::checkForEmptyValue($url);
```

Comment:
To preserve compatibility with CSV, validation should be moved to `getDomSubtree`, because CSV tolerates relative URLs, which XML doesn't.